lib/core/src/lib/auth/services/jwt-helper.service.spec.ts

```diff
@@ -44,96 +59,94 @@ describe('JwtHelperService', () => {
         });
-   });
+    });
 
     describe('ClientRole ', () => {
-
         it('Should be true if the resource_access contains the single role', () => {
-            spyOn(jwtHelperService, 'getAccessToken').and.returnValue('my-access_token');
-
-            spyOn(jwtHelperService, 'decodeToken').and.returnValue(
-                {
-                    resource_access: { fakeApp: { roles: ['role1'] } }
-                });
+            spyOn(jwtHelperService, 'decodeToken').and.returnValue({
+                resource_access: { fakeApp: { roles: ['role1'] } }
+            });
 
             const result = jwtHelperService.hasRealmRolesForClientRole('fakeApp', ['role1']);
             expect(result).toBeTruthy();
         });
 
         it('Should be true if the resource_access contains at least one of the roles', () => {
-            spyOn(jwtHelperService, 'getAccessToken').and.returnValue('my-access_token');
-
-            spyOn(jwtHelperService, 'decodeToken').and.returnValue(
-                {
-                    resource_access: { fakeApp: { roles: ['role1'] } }
-                });
+            spyOn(jwtHelperService, 'decodeToken').and.returnValue({
+                resource_access: { fakeApp: { roles: ['role1'] } }
+            });
 
             const result = jwtHelperService.hasRealmRolesForClientRole('fakeApp', ['role1', 'role2']);
             expect(result).toBeTruthy();
         });
 
         it('Should be false if the resource_access does not contain the role', () => {
-            spyOn(jwtHelperService, 'getAccessToken').and.returnValue('my-access_token');
-            spyOn(jwtHelperService, 'decodeToken').and.returnValue(
-                {
-                    resource_access: { fakeApp: { roles: ['role3'] } }
-                });
+            spyOn(jwtHelperService, 'decodeToken').and.returnValue({
+                resource_access: { fakeApp: { roles: ['role3'] } }
+            });
             const result = jwtHelperService.hasRealmRolesForClientRole('fakeApp', ['role1', 'role2']);
             expect(result).toBeFalsy();
         });
 
         it('Should be false if the resource_access does not contain the client role related to the app', () => {
-            spyOn(jwtHelperService, 'getAccessToken').and.returnValue('my-access_token');
-            spyOn(jwtHelperService, 'decodeToken').and.returnValue(
-                {
-                    resource_access: { anotherFakeApp: { roles: ['role1'] } }
-                });
+            spyOn(jwtHelperService, 'decodeToken').and.returnValue({
+                resource_access: { anotherFakeApp: { roles: ['role1'] } }
+            });
             const result = jwtHelperService.hasRealmRolesForClientRole('fakeApp', ['role1', 'role2']);
             expect(result).toBeFalsy();
         });
-   });
+    });
+});
+
+describe('JwtHelperService with custom storage service', () => {
+    let jwtHelperService: JwtHelperService;
+
+    beforeEach(() => {
```

Comment:
Wouldn't this unit test add more value if we save/read the element on the storage with a complete different key?